pkg/monitors/nagios/nagios.go

```diff
@@ -38,6 +38,12 @@ type Config struct {
 	Service string `yaml:"service" validate:"required"`
 	// The max execution time allowed in seconds before sending SIGKILL
 	Timeout int `yaml:"timeout" default:"9"`
+	// If `false` and change is detected on `stdout` compared to the last
+	// event it will send a new one
+	FilterStdOut bool `yaml:"filterStdOut" default:"false"`
```

Comment:
I would call this `IgnoreStdOut`, etc.  "Filter" is not very clear.

Reply:
done